src/main.js

```diff
@@ -77,35 +83,28 @@ router.beforeEach((to, from, next) => {
     next(path);
     return;
   }
-  if (to.path == '/_=_') {
+  if (to.path == "/_=_") {
     return next(false);
   }
   if (to.fullPath.substr(-1) !== "/") {
-    const path = to.fullPath + "/"
+    const path = to.fullPath + "/";
     next(path);
     return;
   }
-  
-  next();
-})
 
-import { firestorePlugin } from 'vuefire'
-import { rtdbPlugin } from 'vuefire'
-
-
-Vue.use(firestorePlugin)
-Vue.use (rtdbPlugin)
+  next();
+});
 
 var vm = new Vue({
-  el: '#app',
+  el: "#app",
   router,
   render: h => h(App),
   //mounted: () => document.dispatchEvent(new Event("x-app-rendered")),
-  data () {
+  data() {
     return {
       gSheet: null
-    }
+    };
   }
-})
+});
 
-global.vm = vm;
\ No newline at end of file
+global.vm = vm;
```

Comment:
This file mainly seems to show formatting / linting differences

But also you ripped out vuefire, and some other imports now happen elsewhere